lib/src/components/DatePicker/index.tsx

```diff
@@ -165,11 +165,12 @@ export const DatePicker = forwardRef<'input', DatePickerProps>(
         popperProps={popperProps}
         renderCustomHeader={(props: CustomHeaderProps) => (
           <CustomHeader
-            endYear={endYear}
             isMonthYearPicker={showMonthYearPicker}
             locale={locale}
-            startYear={startYear}
             {...props}
+            // Override props values with our custom ones
+            endYear={endYear}
+            startYear={startYear}
```

Comment:
This is actually a breaking change 🤔 I have some tests that fails for this reason: https://cloud.cypress.io/projects/9wi68x/runs/3490/overview/ebe04664-a3b1-4529-a457-77a6b6277f95/replay?roarHideRunsWithDiffGroupsAndTags=1&ts=1745338263757.4&att=1